intro-to-nfts/intro-to-nfts/README.md

```diff
@@ -0,0 +1,11 @@
+name: Intro to NFTs
```

Comment:
Since the topic is named "Intro to NFT's" let's name the Course something different like
```suggestion
name: Introduction
```